### ISSUE TITLE
[Feature Request]: Add custom font

### ISSUE THREAD

**Issue by rt4914** [rt4914/KlinicCon-Frontend#28]
### Is your feature request related to a problem? Please describe.

Out website need custom font.

### Describe the solution you'd like

Add Open-sans as default font for website https://fonts.google.com/specimen/Open+Sans

### Describe alternatives you've considered

Gorditta font is getting used in website but its a paid font so as a replacement we are using Open-sans

**Comment by KvngJamesNG** [comment]
Observation:
I noticed Gordita font is not paid using https://fonts.google.com/specimen/Open+Sans provided.

Describe alternatives you've considered:
-Using CSS preprocessor but we might not be using multiple font types.
-Using a CDN service which might reduce the load time of the project or even create invisible texts.
-Using Google Fonts API with Fallbacks (this is more useful if we want to create backup font for this project, incase the default one fails)
-Using Self-hosting the Font Files 
I decided to go for the last option for numbers of reasons like the font is part of the project directly and can be manipulated the way we want.
Also, this method improves control over font loading performance and ensures that your project isn’t dependent on an external service like Google Fonts, CDN.
